youtube_dl/extractor/eporner.py

```diff
@@ -16,7 +16,7 @@
 
 
 class EpornerIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?eporner\.com/(?:hd-porn|embed)/(?P<id>\w+)(?:/(?P<display_id>[\w-]+))?'
+    _VALID_URL = r'https?://(?:www\.)?eporner\.com/(?:hd-porn|embed|video-)/?(?P<id>\w+)(?:/(?P<display_id>[\w-]+))?'
```

Comment:
Incorrect.

Reply:
Sorry, i don't have skills yet, please see the recent one.